include/wx/stc/stc.h

```diff
@@ -5197,7 +5200,8 @@ class WXDLLIMPEXP_STC wxStyledTextCtrl : public wxControl,
                              const wxColour& currentBgColour = wxNullColour,
                              const wxColour& currentTextColour = wxNullColour);
 
-
+    //Show autocompletion with a more intelligent way
```

Comment:
This comment is not really helpful... I have no idea what does it really mean.

Reply:
it was for my personal use and forgot to zap it off